util/bench/bench_runner.rs

```diff
@@ -0,0 +1,26 @@
+use std::process;
+
+fn main() {
+    let mut cmd = process::Command::new(env!("BENCH_BINARY"));
+
+    // Get args but skip the binary
+    let args: Vec<String> = std::env::args().skip(1).collect();
+
+    // If there are arguments passed to the binary, use those
+    // otherwise, use that the binary was compiled with.
+    if !args.is_empty() {
+        cmd.args(args);
+    } else {
+        cmd.args(env!("BENCH_ARGS").split(" "));
```

Comment:
Could we use https://crates.io/crates/shlex here? I know dependencies here are fiddly, but the crate itself has no dependencies, so it's just n=1...

Reply:
I'd like to avoid using external dependencies if possible. There's no prior art for it and if I was going to add one at all, I'd love to this rule to have a built in way of running `criterion` instead of just hoping users know how set that up themselves. Instead, could we try to get something mostly working and free of dependencies and then discuss how to add them so we have that option readily available should an issue come up?